### PR TITLE
Branch 4

### PR DESCRIPTION
**Pull request by mkpi-pm**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+  getArticleById,
+} = require("./db/controllers/news.controller");
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  console.log(err.code);
```
Comment (scar1377) [inline app.js:14]: remove console.log()

File: db/models/news.model.js [hunk 2]
```diff
@@ -2,7 +2,17 @@ const db = require("../connection");
 
 exports.fetchTopics = () => {
   return db.query("SELECT * FROM topics").then((response) => {
-    // console.log(response.rows);
```
Comment (scar1377) [inline db/models/news.model.js:5]: remove console.log()

**Review by scar1377 — COMMENTED** [review]
Code looks good. Just a couple of console.logs to remove, then it's good to merge to main.

File: app.js (hunk 1 above)
Comment (mkpi-pm) [reply app.js:14]: Thank you! Should I ask for pr again after removing of consoles?

**Pull request merged**